docs/user/alerting/stack-alerts/index-threshold.asciidoc

```diff
@@ -51,68 +48,53 @@ When the index threshold alert condition is met, the following variables are ava
 
-Click on each clause to open a control that helps you set the value:
-
-[float]
-==== Index clause
-The index clause control will list and allow you to search for available indices. Choose *kibana_sample_data_logs*
-
+. Click *Index*, and set *Indices to query* to *kibana_sample_data_logs*.
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-example-index.png[Choosing an index]
 
-Once an index is selected, the list of time fields for that index will be available to select. Choose *@timestamp*.
-
+. Set the *Time field* to *@timestamp*.
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-example-timefield.png[Choosing a time field]
 
-[float]
-==== When clause
-
-We want to detect the number of bytes served during the time window, so we select `sum` as the aggregation, and `bytes` as the field to aggregate. 
-
+. To detect the number of bytes served during the time window, click *When* and select `sum` as the aggregation, and bytes as the field to aggregate.
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-example-aggregation.png[Choosing the aggregation]
 
-[float]
-==== Over/Grouped over clause
-
-We want to alert on the three sites that have the most traffic, so we'll group the sum of bytes by the `host.keyword` field and take the top 3 values. 
-
+. To detect the four sites that have the most traffic, click *Over* and select `top`, enter `4`, and select `host.keyword` as the field.
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-example-grouping.png[Choosing the groups]
 
-[float]
-==== Threshold clause
-
-We want to alert when any site exceeds 420,000 bytes over a 24 hour period, so we'll set the threshold to 420,000 and use the `is above` comparison. 
-
+. To alert when any of the top four sites exceeds 420,000 bytes over a 24 hour period, select `is above` and enter `420000`.
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-example-threshold.png[Setting the threshold]
 
-[float]
-==== Time window clause
-
-Finally, set the time window to 24 hours to complete the alert configuration.
-
+. Finally, click *For the last*, enter `24` and select `hours` to complete the alert configuration.
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-example-window.png[Setting the time window]
 
-The preview chart will render showing the 24 hour sum of bytes at 4 hours intervals (the *check every* interval) for the past 120 hours (the last 30 intervals).
-
+. The preview chart will render showing the 24 hour sum of bytes at 4 hours intervals (the *check every* interval) for the past 120 hours (the last 30 intervals).
++
 [role="screenshot"]
 image::user/alerting/images/alert-types-index-threshold-example-preview.png[Setting the time window]
-
-[float]
-==== Comparing time windows
```

Comment:
Can this be step 11?

11.  Change the time window and observe the effect it has on the chart. Compare ....